src/extension-runners/chromium.js

```diff
@@ -127,8 +127,12 @@ export class ChromiumExtensionRunner {
       chromeFlags.push(...this.params.args);
     }
 
+    let userDataDir = false;
     if (this.params.chromiumProfile) {
-      chromeFlags.push(`--user-data-dir=${this.params.chromiumProfile}`);
+      const pathParts = path.parse(this.params.chromiumProfile);
+      const profileDir = pathParts.base;
+      chromeFlags.push(`--profile-directory=${profileDir}`);
```

Comment:
Yes exactly! I changed it this way to have it fully compatible with --firefox-profile, so that the user provides the full path to the profile, and not the folder containing the profiles. This way the user also doesn't have to deal with the undocumented --profile-directory flag. The documentation of --chromium-profile ("Path to a custom Chromium profile") remains correct this way.

If the default --user-data-dir is correct, it is also possible to provide only the name of the profile, like for Firefox.

Do you have concerns?

Reply:
To simplify @Rob--W , `--chromium-profile = --user-data-dir + --profile-directory`

Reply:
For the most common scenarios (using a temporary directory, or even a copy of a single-user profile), just passing `--user-data-dir` is sufficient. I don't want to complicate `web-ext` with `--profile-directory`, because this is a special case AND already possible without special support from `web-ext` (i.e. the `--arg` flag).

Reply:
Okay for me @Rob--W, but it will mean:
- change the [documentation](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/) of `--chromium-profile` from "Path to a custom Chromium profile." to "Path to the directory containing the profiles" (or equivalent). Can you organize this change?
- loose the compatibility with `--firefox-profile`
I don't agree with it, but if you insist I do the change since I would like to have it finally somehow working.

Reply:
There are two bugs here:

* According to your comment at https://github.com/mozilla/web-ext/issues/1909#issuecomment-637594465 , the `--chromium-profile` flag fails to match the specified directory on Windows (even though it works fine on Linux). This is a bug that we should certainly fix.
* Now in this PR and thread, you are advocating for support of a specific "User Account" within a given user data directory, via the `--profile-directory` flag. This is more work than needed to fix the previous bug, and I didn't feel the need to delay the fix of the previous bug for this new feature.

I'm not adamant against supporting User Accounts, but note that it will take some effort. I'll post a separate comment with lots of more details, I hope that it helps.